bugsnag/utils.py

```diff
@@ -81,13 +81,15 @@ def default(self, obj):
             bugsnag.logger.exception('Could not add object to payload')
             return self.unencodeable_value
 
-    def _sanitize(self, obj, trim_strings, ignored=None):
+    def _sanitize(self, obj, trim_strings, ignored=None, seen=None):
         """
         Replace recursive values and trim strings longer than
         MAX_STRING_LENGTH
         """
         if not ignored:
             ignored = set()
+        if not seen:
+            seen = []
```

Comment:
Consider adding a comment to explain why this `seen` parameter is needed, as it probably isn't obvious to someone reading the code without looking at the pull request description or commit message.